test/extended/cli/explain.go

```diff
@@ -193,6 +193,13 @@ var (
 		{Group: "performance.openshift.io", Version: "v2", Resource: "performanceprofiles"},
 	}
 
+	microshiftCRDTypes = []schema.GroupVersionResource{
+		{Group: "route.openshift.io", Version: "v1", Resource: "routes"},
+		{Group: "topolvm.io", Version: "v1", Resource: "logicalvolumes"},
+		//{Group: "security.internal.openshift.io", Version: "v1", Resource: "rangeallocations"},
+		//{Group: "security.openshift.io", Version: "v1", Resource: "securitycontextconstraints"},
+	}
```

Comment:
`oc explain` for these two resources do not have `spec` and `status` fields, and the targeted test `[sig-cli] oc explain should contain proper spec+status for CRDs` is meant to check `spec` and `status`. 

These two resources are already present in `specialTypes` list and are covered separately in another test with specific `field` and `pattern`. So, there should not be any problem if we don't keep them again here.

Reply:
A comment about that in the code would help future maintainers.

You say the 2 CRDs do not have spec or status fields. Shouldn't they? Does the fact that they are missing indicate some sort of problem?

Reply:
Added a comment 

The missing fields might be by design, as they are not present in OpenShift also. So, I think this is not a problem.